src/Parser/Mage/Frost/Modules/Items/ShatteredFragmentsOfSindragosa.js

```diff
@@ -0,0 +1,33 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+class ShatteredFragmentsOfSindragosa extends Module {
+
+  static dependencies = {
+		combatants: Combatants,
+	};
+
+  damage = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasHead(ITEMS.SHATTERED_FRAGMENTS_OF_SINDRAGOSA.id);
+  }
+
+  on_byPlayer_damage(event) {
+    if (event.ability.guid !== SPELLS.COMET_STORM_DAMAGE.id) {
+      return;
+    }
+      this.damage += event.amount;
```

Comment:
You should add absorbed damage as well (Maiden, Avatar etc.) So `this.damage += event.amount + (event.absorbed || 0);`. Actually I'm never sure if it's absorb or absorbed, check to make sure.

Reply:
Oops, forgot to add that. Should be added now